bika/lims/exportimport/dataimport.py

```diff
@@ -121,12 +123,17 @@ def __call__(self):
         # If a specific template for this instrument doesn't exist yet,
         # use the default template for instrument results file import located
         # at bika/lims/exportimport/instruments/instrument.pt
-        import os.path
-        instrpath = os.path.join("exportimport", "instruments")
-        templates_dir = resource_filename("bika.lims", instrpath)
-        fname = "%s/%s_import.pt" % (templates_dir, exim)
+        if exim.startswith('senaite/instruments'):
+            #TODO find a better way to see check if the instrument is in core
```

Comment:
Yes, indeed. Not fancy to have `senaite/instruments` hard-coded here. But since (correct me if I am wrong), the plan is to move all these instrument interfaces to `senaite.instruments`, then we can consider this as a "temporary" thingy until all them get ported.

Reply:
I @xispa, I created a method to better determine is the instrument import interface is in or out of core. See latest commit